tests/DOCUMENTATION.md

```diff
@@ -0,0 +1,36 @@
+** #Documentation **
```

Comment:
The name of this file should better reflect the topic of this documentation. Something that points to test cases definition, or the like. DOCUMENTATION.md is too broad